pages/datasets/_datasetId.vue

```diff
@@ -107,6 +107,17 @@
           <button class="dataset-button" @click="isDownloadModalVisible = true">
             Get Dataset
           </button>
+          <nuxt-link
+            :to="{
+              name: 'help-helpId',
+              params: {
+                helpId: '3FXikFXC8shPRd8xZqhjVT'
```

Comment:
Is this the Contentful Id? If so, do you have a config file somewhere that you have references to all the model instance ids? It might not be a bad idea to keep these centrally located and you just reference a variable in the event you wish to ever swap these out.

Reply:
Definitely. I moved this to our `nuxt.config.js` file.